include/nccl-headers/nvidia/err.h

```diff
@@ -11,6 +11,7 @@ typedef enum { ncclSuccess                 =  0,
                ncclSystemError             =  2,
                ncclInternalError           =  3,
                ncclInvalidArgument         =  4,
+               ncclInvalidUsage            =  5,
```

Comment:
We've always kept these exactly as in the ext-net directory.  We should document why we need to do otherwise and talk to the NCCL team about fixing this.